### DISCUSSION TITLE
sserialsend - how to include <CR><LF> Characters

### DISCUSSION THREAD

**Discussion by letshin** [arendst/Tasmota#21806] [Q&A] [answered]
Hi. I'm interfacing with a sensor and the datasheet states 
`"All UART commands must be terminated with a carriage return and line feed <CR><LF>, hex 0x0D 0x0A". `
I've tried sending this as `sserialsend .\r\n` but the sensor returns a '?' which the datasheet states that means the command is not recognised. I've also tried `sserialsend . 0x0D 0x0A` with the same result.

Can anyone let me know what's the best way to do this please. Thanks.

**Comment by sfromis** [comment, marked as answer]
If you want to have escape characters like `\r\n` replaced, you should use the variant `SSerialSend3` which explicitly enables this, as of the docs about the `SSerialSend` command.
